ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -86,12 +107,16 @@ const ConfirmButton = ({
           block
           danger
           data-testid="confirm-footer-button"
-          disabled={hasUnconfirmedDangerAlerts ? false : disabled}
+          disabled={getButtonDisabledState(
+            hasUnconfirmedDangerAlerts,
+            hasDangerBlockingAlerts,
+            disabled,
+          )}
```

Comment:
Above condition looks bit unclear to me if `hasDangerBlockingAlerts` is true disabled will always be false.

Reply:
If the `hasDangerBlockingAlerts` disabled will always be true, so the button will be disabled until the user resolves the blocking alert in the current case this is the insufficient balance alert.